src/pyhub/llm/agents/tools/code_interpreter/backends/local.py

```
@@ -0,0 +1,363 @@
+"""Local execution backend with restricted environment."""
+
+import io
+import os
+import sys
+import time
+import traceback
+from contextlib import contextmanager, redirect_stdout, redirect_stderr
```

Ruff (0.11.9)

`contextlib.redirect_stdout` imported but unused

Remove unused import

(F401)